views/bulkAppointmentsAddDetails.njk

```diff
@@ -101,7 +101,7 @@
                         label: {
                             text: "Appointment type"
                         },
-                        items: appointmentTypes | addDefaultSelectedVale('Select') | setSelected(appointmentType),
+                        items: appointmentTypes | filterNot('value', ['VLB', 'VLPM']) | addDefaultSelectedVale('Select') | setSelected(appointmentType),
```

Comment:
Should these codes be pulling out into variables with better names?

Reply:
I think its fine as is. This code is (hopefully) short-lived.